rest-api-spec/src/main/resources/rest-api-spec/test/indices.get_mapping/70_mix_typeless_typeful.yml

```diff
@@ -0,0 +1,23 @@
+---
+"GET mapping with typeless API on an index that has types":
```

Comment:
Should we also backport the test changes to the standard rest tests (`10_basic`, `11_basic_with_types`, etc.)?

Reply:
Also pushed a commit to add these tests.